### ISSUE TITLE
22929209: List of UI locations where Darken Colours SHOULD BE applied BUT IS NOT

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#6273]
#### Description

Summary:
The [Darken Colours] setting, which is available under [General > Accessibility > Increase Contrast], is not uniformly applied across all the appropriate UI buttons and widgets; this has been the case since iOS 9.0 and persists in iOS 9.0.2 as well.

Full list of noticed UI widget locations where Darken Colours effect should be applied but is not, is provided in the "Steps to reproduce" section below.

Steps to Reproduce:
Under: Settings > General:
Language & Region > Other Languages...
Storage & iCloud Usage > iCloud > Manage Storage > Change Storage Plan
Restrictions > Enable Restrictions
Date & Time > Set Automatically [is Off] > Time display
VPN > Add VPN Configuration...
Reset > (All Buttons)

Under: Settings > Wi-Fi:
Tap (i) button for a connected Network > Forget This Network
Tap (i) button for a connected Network > Renew Lease

Under: Settings > Touch ID & Passcode:
Add Fingerprint...
Turn Passcode Off
Change Passcode

Under: Settings > Privacy:
Advertising > Reset Advertising Identifier...

Under: Settings > iCloud:
Set up Family Sharing...
Storage > Buy More Storage

Under: Settings > Apps and iTunes Stores:
Apple ID: johnappleseed@icloud.com

Under: Settings > Mails, Contacts, Calendars:
Add Account > Other
iCloud > Create new Apple ID
Import SIM Contacts

Under: Settings > Phone:
Change Voicemail Password
SIM Applications > (All Buttons)
(Mobile Network Shortcuts e.g. Vodafone, O2 Home, etc.)
Blocked > Add New...

Under: Settings > iMessage:
Send & Receive > Add an Email...

Under: Settings > FaceTime:
Add an Email...
Blocked > Add New...

Under: Settings > Safari:
Advanced > Website Data > Show All Sites
Clear History and Website Data

Under: Settings > Music:
Subscribe to iTunes Match
(Home Sharing section) > Sign In

Under: Settings > Videos:
(Home Sharing section) > Sign In

Under: Settings > Game Center:
Sign In
Forgot Apple ID or Password?
Export Logs

Under: Settings > Twitter:
Create New Account
Learn More about Twitter

Under: Settings > Facebook:
Create New Account
Learn More about Facebook

Under: Settings > Vimeo:
(put some dummy sign-in details) > Sign In
Learn More About Vimeo

Under: Settings > Developer
Reset Media Services

Under: Messages > New iMessage:
(a) "Send" button
(b) SMS / iMessage contacts in lookup list don't have effect applied (dark green / blue respectively)

Under: Messages > [Select any Contact here...] > Details:
Send My Current Location
Share My Location

Expected Results:
Above mentioned locations should look darker as per Darken Colours effect.

Actual Results:
Standard colour contrast (light blue, light green, etc.)

Version:
iOS 9.0, iOS 9.0.1, iOS 9.0.2

Notes:

Configuration:
Noticed when Settings > General > Accessibility > Increase Contrast > Darken Colours is turned ON.

Attachments:
## 

Product Version: 9.0, 9.0.1, 9.0.2
Created: 2015-10-01T07:04:00.186200
Originated: 2015-01-10T00:00:00
Open Radar Link: http://www.openradar.me/22929209
